src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java

```diff
@@ -70,6 +72,7 @@ public void setUp() throws IOException {
         when(mockYouTubeClient.channels()).thenReturn(mockChannels);
         when(mockChannels.list("contentDetails")).thenReturn(mockListChannels);
         when(mockListChannels.setId(anyString())).thenReturn(mockListChannels);
+        when(mockListChannels.setForUsername(anyString())).thenReturn(mockListChannels);
```

Comment:
instead of anySTring() use the actual USER_NAME.   same with the previous line instead of anyString() put CHANNEL_ID.
(this ensures if some other argument got passed it returns null instead of an actual object.).

Reply:
Compiler error.  This is because I also pass in a NONEXISTENT_CHANNEL_ID / NONEXISTENT_USERNAME as a parameter.